paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -85,14 +85,16 @@ void CPUQuantizePass::QuantizeInput(Graph* g,
                                     std::string scale_attr_name,
                                     float shift,
                                     std::string shift_attr_name) const {
-  auto inputs = op->Op()->InputNames();
+  bool residual_fc = input_name == "ResidualData" && op->Op()->Type() == "fc";
+  auto inputs = residual_fc ? op->Op()->OutputNames() : op->Op()->InputNames();
   bool name_found =
       std::find(inputs.begin(), inputs.end(), input_name) != inputs.end();
   PADDLE_ENFORCE_EQ(name_found,
                     true,
                     platform::errors::InvalidArgument(
-                        "Var(%s) isn't the input of the %s operator.",
+                        "Var(%s) isn't the %s of the %s operator.",
                         input_name,
+                        residual_fc ? "output" : "input",
```

Comment:
It's stinky for me, what if we will add more ops with residual connection in the future ?

Reply:
It depends on what we will agree on with baidu. Previously it was easy because we assumed that ResidualData will be input because on the graph it's linked as input and in OpDesc it was set as input. But recently when FC with residual was added, we were instructed to use Output in OpDesc but it's impossible to set it as output in graph so it's still linked as input. From this comes this confusion in code.